src/CMakeLists.txt

```diff
@@ -16,12 +16,8 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 
 # find HTSEngine
 # adapted from https://github.com/r9y9/open_jtalk/blob/1.10/src/CMakeLists.txt
-find_path(HTS_ENGINE_INCLUDE_DIR HTS_engine.h)
-find_library(HTS_ENGINE_LIB hts_engine_API)
-if(NOT HTS_ENGINE_LIB)
-  # fallback
-  find_library(HTS_ENGINE_LIB HTSEngine)
-endif()
+
+add_subdirectory(lib/hts_engine_API/hts_engine/src)
```

Comment:
Is it possible to revert this change? It would be useful to have it as a shared library so that the other libraries (such as open_jtalk) can use it as well.

Reply:
Or adding an option to embed hts_engine_API source as well.

Reply:
I think the issue is that Windows doesn't have a native package manager like linux has so it's a bit harder to handle global dependencies, if open_jtalk uses hts_engine, i think it could also use it as a submodule

Reply:
what do you think would be best ?

Reply:
I am so sorry for the late reply. I was wondering if we want to make the static build a configurable option. That said, I am okay with the change. if you are okay I will merge this PR.

Reply:
actually it is totally possible to make hts_engine_api a shared library instead of a static library

Reply:
Thanks. I am gonna merge ths PR!